mousebender/simple.py

```diff
@@ -72,6 +72,16 @@ class UnsupportedMIMEType(Exception):
 _Meta_1_0 = TypedDict("_Meta_1_0", {"api-version": Literal["1.0"]})
 _Meta_1_1 = TypedDict("_Meta_1_1", {"api-version": Literal["1.1"]})
 _Meta_1_3 = TypedDict("_Meta_1_3", {"api-version": Literal["1.3"]})
+_Meta_1_4 = TypedDict(
```

Comment:
This is wrong. Look at lines 72 through 74; this global variable is just for declaring the "api-version" and its literal.